storage_service/locations/models/package.py

```diff
@@ -289,6 +291,61 @@ def _update_quotas(self, space, location):
         location.used += self.size
         location.save()
 
+    def move_to_location(self, destination_location_uuid):
```

Comment:
Please add a docstring summarizing what this function is doing. If you're not sure of formatting, https://www.python.org/dev/peps/pep-0257/ is a good description.